src/Microsoft.AspNetCore.Mvc.Razor/RazorPage.cs

```diff
@@ -30,10 +31,9 @@ public RazorPage()
         }
 
         /// <summary>
-        /// Gets the <see cref="ITempDataDictionary"/> from the <see cref="ViewContext"/>.
+        /// An <see cref="HttpContext"/> representing the current request execution.
         /// </summary>
-        /// <remarks>Returns null if <see cref="ViewContext"/> is null.</remarks>
-        public ITempDataDictionary TempData => ViewContext?.TempData;
```

Comment:
It got moved to the base: https://github.com/aspnet/Mvc/pull/6034/files#diff-a4f1d91073db434f385e8192d601f7cdR106